src/types.ts

```diff
@@ -8,7 +8,7 @@ export interface Config {
 export interface Block {
   source: string;
   timestamp: number;
-  block_leve: number;
+  block_level: number;
```

Comment:
Może w ogóle warto przy okazji zamienić "block_level" na camelCase'a? "blockLevel" byłoby bardziej spójne z resztą. Ale to już tylko taka dodatkowa sugestia. Wtedy będzie można usunąć komentarz dla linta, który jest nad tym interfejsem, bo lint sprawdza, czy piszemy camelCasem ;)

Reply:
Tu chyba tak wrzucone bylo, bo api CounseilJS zwracalo w takim formacie dane.